src/rust/engine/src/context.rs

```diff
@@ -258,6 +258,10 @@ impl Core {
         docker::ImagePullPolicy::OnlyIfLatestOrMissing,
       )?);
 
+      let runner =
+        SwitchedCommandRunner::new(docker_runner, runner, |req| req.docker_image.is_some());
+      let runner = Box::new(runner) as _;
```

Comment:
Why not directly call `Box::new()` and do this in one line?

Reply:
Beats me. Will change.

Reply:
Fixed.